src/NHibernate/HibernateException.cs

```diff
@@ -12,6 +13,7 @@ namespace NHibernate
 	[Serializable]
 	public class HibernateException : Exception
```

Comment:
I am not for adding this to the hibernate exception base class. If we add that, it should be on a specialized class, like adding `class HibernateAggregateException : HibernateException`.